lusidtools/cocoon/cocoon_printer.py

```diff
@@ -10,12 +10,22 @@ def check_dict_for_required_keys(
     target_dict: dict, target_name: str, required_keys: list
 ):
     """
-    This function checks that a named dictionary contains a list of required keys
-    :param dict target_dict: Dictionary to check for keys in
-    :param str target_name: Variable name of dictionary
-    :param list required_keys: List of required keys
-    :return:
+    This function checks that a named dictionary contains a list of required key
+
+    Parameters
+    ----------
+    target_dict : dict
+        Dictionary to check for keys in
+    target_name : str
+        Variable name of dictionary
+    required_keys : list
+        List of required keys
+
+    Returns
+    -------
+
```

Comment:
What was your determination on how to handle functions that don't return anything?

Reply:
```python
"""
Returns
--------
None
"""
```